server/emails/templates/DocumentMentionedEmail.tsx

```diff
@@ -74,6 +132,15 @@ Open Document: ${teamUrl}${document.url}
             {actorName} mentioned you in the document{" "}
             <a href={documentLink}>{document.title}</a>.
           </p>
+          {body && (
+            <>
+              <EmptySpace height={20} />
+              <Diff>
```

Comment:
Do you mind changing the style of `<Diff>` so that it uses a border instead of a background like this:

<img width="660" alt="image" src="https://github.com/user-attachments/assets/9efc3f5c-bdbf-48a2-8149-97f723110d22">

It will help to make mentions display correctly

Reply:
Just for the mention (or) the diff element in itself?

This styling of diff looks neat, imo.

Reply:
I was thinking for diff element itself, the grey background clashes with mentions and quotes so you can't see them properly

Reply:
Looks weird with too many lines for tables..
![Screenshot 2024-10-11 at 10 34 23 PM](https://github.com/user-attachments/assets/79120bae-c0ec-4acf-91ee-2195faca094a)

How does this look?
![Screenshot 2024-10-11 at 10 39 15 PM](https://github.com/user-attachments/assets/1be2cb86-e45f-4423-ab95-6bf8fbd23f1e)

Reply:
I'm ok with the table case, it will be quite rare to just show up like this - changing the background color just has a few too many edgecases. 

I don't know if we could do something to even up the padding that would help a lot